src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+promise1.then((message) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = message;
+
+  document.body.appendChild(div);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
```

Comment:
Consider adding a `.finally()` block to handle any cleanup or final actions that should occur regardless of whether the promise resolves or rejects. This can improve code robustness.